app/grandchallenge/algorithms/migrations/0065_create_algorithm_interfaces.py

```diff
@@ -17,6 +17,10 @@ def create_algorithm_interfaces(apps, _schema_editor):
         inputs = algorithm.inputs.all()
         outputs = algorithm.outputs.all()
 
+        if not inputs or not outputs:
+            # Skip algorithms without inputs or outputs
+            continue
+

```

Comment:
I realized that this checks is necessary since the migration cannot use the custom create method on the manager (where I added this check in the last PR).

Reply:
Do we have such algorithms? Would be very strange if so, I think it is/should be forbidden.

Reply:
We should not, but better to check. For phases, this can happen since those used to be configured in the admin and there was no check for whether inputs and outputs are both present when converting to algorithm submission type.